pkg/routeagent_driver/handlers/ovn/config_syncer.go

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Does restart the relevant pod will fix this issue? 

If the answer is yes, I wonder if we can handle this issue by adding liveness probe to the pod ?

Reply:
as discusses this a idea we need to explore and could this of doing that in next release for multiple use-cases.